SU2_CFD/src/solvers/CSolver.cpp

```diff
@@ -1650,6 +1650,10 @@ void CSolver::GetCommCountAndType(const CConfig* config,
       COUNT_PER_POINT  = nDim;
       MPI_TYPE         = COMM_TYPE_DOUBLE;
       break;
+    case AXIAUXVAR_GRADIENT:
+      COUNT_PER_POINT  = nDim*3;
```

Comment:
The 3 is a little bit magic, I would define it somewhere central like CVariable.hpp as a constexpr variable.

Reply:
Im not sure I follow here

Reply:
Ideally this kind of info should be centralized somewhere, typically in option_structure.hpp.
e.g. const unsigned short NAXIVARS=3; or something that follows similar names we might have.
Then you use that variable in the code instead of hardcoding 3 in lots of places (3 can be confused with 3D, and if you ever need more variables...)

Reply:
Ah i gotcha, that makes sense!